scripts/lint-style.py

```diff
@@ -254,7 +254,7 @@ def regular_check(lines, path):
         if copy_done and line == "\n":
             continue
         words = line.split()
-        if words[0] != "import" and words[0] != "/-!" and words[0] != "#align_import":
+        if words[0] != "import" and words[0] != "--" and words[0] != "/-!" and words[0] != "#align_import":
```

Comment:
I'm not sure this should this be part of this PR, but if you think it's OK I'll let it slide.

I think this is a good first version of the tactic, and I'm happy if you merged it.

bors d+